### ISSUE TITLE
Then the "something" button should be disabled

### ISSUE THREAD

**Issue by johngeorgewright** [makandra/spreewald#18]
https://github.com/makandra/spreewald/blob/master/lib/spreewald/web_steps.rb#L588

When a button is disabled, it can't be found. Looks like [as of Capybara 2.1.0 you can use the `disabled` flag](https://github.com/jnicklas/capybara/issues/982) when searching for elements.

**Comment by denzelem** [comment]
This should fix the issue. There seem to be no test present for this step yet. PR required.

```
# Tests that an input or button with the given label is disabled.
Then /^the "([^\"]*)" (field|button) should( not)? be disabled$/ do |label, kind, negate|
  if kind == 'field'
    element = find_field(label, disabled: :all)
  else
    element = find_button(label, disabled: :all)
  end
  ["false", "", nil].send(negate ? :should : :should_not, include(element[:disabled]))
end.overridable
```

**Comment by FLeinzi** [comment]
We fixed that issue in version 2.2.2
